app/controllers/classifications_controller.rb

```diff
@@ -1,5 +0,0 @@
-class ClassificationsController < PublicFacingController
```

Comment:
With this commit message, could it be made slightly clearer that it is the classification and topical event controllers/tests/routes that are being removed.